### ISSUE TITLE
Error response from daemon: mkdir /etc/localtime: file exists

### ISSUE THREAD

**Issue by alop** [kubernetes/minikube#1324]
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Minikube version** (use `minikube version`): v0.17.1

**Environment**:
- **OS** (e.g. from /etc/os-release): Mac OS Sierra
- **VM Driver** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep DriverName`): virtualbox or xhyve
- **ISO version** (e.g. `cat ~/.minikube/machines/minikube/config.json | grep -i ISO` or `minikube ssh cat /etc/VERSION`): v1.0.7
- **Install tools**:
- **Others**:

**What happened**:
When attempting to deploy kolla-kubernetes in minikube, all the pods failed with
Error response from daemon: mkdir /etc/localtime: file exists

**What you expected to happen**:
Should work

**How to reproduce it** (as minimally and precisely as possible):
Deploying kolla-kubernetes is somewhat convoluted, but the issue is also reported at
https://github.com/mattermost/mattermost-docker/issues/106
also
gtirloni [2:20 PM] 
$ docker run -ti -v /etc/localtime:/etc/localtime:ro busybox /bin/sh
docker: Error response from daemon: mkdir /etc/localtime: file exists (edited)

**Anything else do we need to know**:

**Comment by gtirloni** [comment]
This issue is caused by broken symlink `/etc/localtime -> /usr/share/zoneinfo/UTC`.

I think I have a good idea of what the fix might look like. Currently building a new ISO image to test this.

**Comment by r2d4** [comment]
This will be fixed in the next release of minikube (0.18), which should be coming soon.

For now, you can use the iso from the CI build by recreating minikube with a `minikube delete` then a 

```
minikube start --iso-url https://storage.googleapis.com/minikube-builds/1327/minikube-testing.iso
```

**Comment by r2d4** [comment]
Closing this, the fix has been merged and you can point to that built image now if you need it before our next release.
